src/index.py

```diff
@@ -52,4 +78,5 @@ def main(config: FunctionConfig) -> Optional[Function]:
     )
 
     if function is not None:
+        # return output parameter
         print(f"::set-output name=function_external_id::{function.external_id}")
```

Comment:
Should this print-statement be changed to logging?

Reply:
yes. that's how you return values out of action. magic syntax :)

Reply:
love it